farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.spec.js

```diff
@@ -56,6 +56,42 @@ describe('Testing for the seeding report page', () => {
 
         
     })
+    context.only('can see spinner at appropriate times', () => {
```

Comment:
Using .only is great for development and testing of our changes. Please remove the .only here and rerun all tests to ensure they still pass.  The commit the version without the .only so that all tests will run in the upstream version.